### PR TITLE
Add to remaining singletons; fixes #11

### LINKED ISSUE #11: Add more keywords to emoji that has only one keyword

### PR DESCRIPTION
**Pull request by ivavictoria**

### CODE REVIEW

File: emojis.json [hunk 1]
```diff
@@ -85,9 +85,9 @@
   "hear_no_evil": ["animal", "monkey", "nature"],
   "speak_no_evil": ["monkey", "animal", "nature", "omg"],
   "skull": ["dead", "skeleton", "creepy"],
-  "alien": ["UFO", "paul"],
+  "alien": ["UFO", "paul", "weird", "space"],
```
Comment (muan) [inline emojis.json]: what about `outer space`?

File: emojis.json [hunk 2]
```diff
@@ -85,9 +85,9 @@
...
   "hankey": ["poop", "shitface"],
-  "poop": ["shitface"],
+  "poop": ["shitface", "bad", "fail", "oops"],
```
Comment (muan) [inline emojis.json]: `oops` is a bit of a stretched. :confused:

File: emojis.json [hunk 3]
```diff
@@ -724,7 +724,7 @@
   "minibus": ["vehicle", "car", "transportation"],
   "video_camera": ["film", "record"],
   "cancer": ["sign", "zodiac", "purple-square", "astrology"],
-  "flower_playing_cards": ["game"],
+  "flower_playing_cards": ["game", "sunset", "red", "deck", "spring"],
```
Comment (muan) [inline emojis.json]: I'd remove `deck` and `spring` from this one.

File: emojis.json [hunk 4]
```diff
@@ -500,7 +500,7 @@
   "pisces": ["purple-square", "sign", "zodiac", "astrology"],
   "taurus": ["purple-square", "sign", "zodiac", "astrology"],
   "bamboo": ["plant", "nature", "vegetable", "panda"],
-  "diamond_shape_with_a_dot_inside": ["jewel"],
+  "diamond_shape_with_a_dot_inside": ["jewel", "blue", "gem", "crystal", "fancy", "pretty"],
```
Comment (muan) [inline emojis.json]: `pretty` seems too broad here.

File: emojis.json [hunk 5]
```diff
@@ -385,41 +385,41 @@
   "aries": ["sign", "purple-square", "zodiac", "astrology"],
   "convenience_store": ["building", "shopping", "groceries"],
   "speedboat": ["ship", "transportation", "vehicle", "summer"],
-  "arrow_backward": ["blue-square"],
+  "arrow_backward": ["blue-square", "left", "point", "direction"],
```
Comment (muan) [inline emojis.json]: I would prefer we do without `point` on all these ones. :grimacing:

**Comment by ivavictoria** [comment]
Points taken! :smile:

**Comment by muan** [comment]
:heart::heart: thanks a bunch!!!

**Pull request merged**